garden.turtle.Jellybean.yml

```diff
@@ -0,0 +1,36 @@
+---
+app-id: garden.turtle.Jellybean
+runtime: org.gnome.Platform
+runtime-version: master
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.vala
+command: jellybean
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--device=dri"
+- "--socket=wayland"
+build-options:
+  append-path: "/usr/lib/sdk/vala/bin"
+  prepend-ld-library-path: "/usr/lib/sdk/vala/lib"
+cleanup:
+- "/include"
+- "/lib/pkgconfig"
+- "/man"
+- "/share/doc"
+- "/share/gtk-doc"
+- "/share/man"
+- "/share/pkgconfig"
+- "/share/vala"
+- "*.la"
+- "*.a"
+modules:
+- name: jellybean
```

Comment:
Fix these it the desktop file, not essential for the build but nice to have

```
esktop-file-validate garden.turtle.Jellybean.desktop 
garden.turtle.Jellybean.desktop: error: value "Jellybean;Inventory;Number;" for key "Categories" in group "Desktop Entry" contains an unregistered value "Jellybean"; values extending the format should start with "X-"
garden.turtle.Jellybean.desktop: error: value "Jellybean;Inventory;Number;" for key "Categories" in group "Desktop Entry" contains an unregistered value "Inventory"; values extending the format should start with "X-"
garden.turtle.Jellybean.desktop: error: value "Jellybean;Inventory;Number;" for key "Categories" in group "Desktop Entry" contains an unregistered value "Number"; values extending the format should start with "X-"
garden.turtle.Jellybean.desktop: hint: value "Jellybean;Inventory;Number;" for key "Categories" in group "Desktop Entry" does not contain a registered main category; application might only show up in a "catch-all" section of the application menu
```

Reply:
issue resolved on the repository end